x-pack/plugins/security_solution/public/entity_analytics/pages/entity_analytics_dashboard.tsx

```diff
@@ -23,16 +23,16 @@ import { RiskScoreUpdatePanel } from '../components/risk_score_update_panel';
 import { useHasSecurityCapability } from '../../helper_hooks';
 import { EntityAnalyticsHeader } from '../components/entity_analytics_header';
 import { EntityAnalyticsAnomalies } from '../components/entity_analytics_anomalies';
+
+import { EntityStoreDashboardPanels } from '../components/entity_store/components/dashboard_panels';
 import { EntityAnalyticsRiskScores } from '../components/entity_analytics_risk_score';
-import { EntitiesList } from '../components/entity_store/entities_list';
 import { useIsExperimentalFeatureEnabled } from '../../common/hooks/use_experimental_features';
 
 const EntityAnalyticsComponent = () => {
   const { data: riskScoreEngineStatus } = useRiskEngineStatus();
   const { indicesExist, loading: isSourcererLoading, sourcererDataView } = useSourcererDataView();
   const isRiskScoreModuleLicenseAvailable = useHasSecurityCapability('entity-analytics');
-
-  const isEntityStoreDisabled = useIsExperimentalFeatureEnabled('entityStoreDisabled');
+  const isEntityStoreFeatureFlagDisabled = useIsExperimentalFeatureEnabled('entityStoreDisabled');
```

Comment:
This seems like a batch text update error on the editor. This feature flag name is not correct. It should be `entityStoreEnabled` as I see in [security_solution/common/experimental_features.ts#L242](https://github.com/elastic/kibana/blob/563910b672b6dbe4f9e7931e36ec41e674fe8eb3/x-pack/plugins/security_solution/common/experimental_features.ts#L242). Also, I would suggest naming the const as a positive query `isEntityStoreFeatureFlagEnabled` instead so we don't mix semantics of `enabled` and `disabled`

Reply:
I think this is correct.
The flag is called `entityStoreDisabled` and is set to `false` by default. 
This was changes in this PR I believe, so maybe the diff is a bit weird?

In any case, the reason for this is that we want to ship the Entity Store in 8.16 as a Technical Preview, enabled by default. But we want to let our users quickly disable it if it causes any issues and/or we get SDHs.